src/suites/positive/positive-suite.ts

```diff
@@ -1,28 +1,53 @@
 import { OperationResult } from '../../validation';
-import { OperationExampleFactory } from '../../oas-parsing/operation-example';
+import {
+  OperationExample,
+  OperationExampleFactory,
+} from '../../oas-parsing/operation-example';
 import { ValidationConductor } from './conductors';
-import Suite from '../suite';
+import Suite, { SuiteConfig } from '../suite';
+import { SecurityValues } from 'swagger-client';
+import { SecurityValuesFactory } from '../../oas-parsing/security-values';
+import OASSchema from '../../oas-parsing/schema';
 
 export default class PositiveSuite extends Suite {
   public static suiteId = 'positive';
 
   protected static label = '(Example Group: 2xx Response)';
 
-  async conduct(): Promise<OperationResult[]> {
-    this.checkTargetServer();
+  private schema!: OASSchema;
+  private server!: string | undefined;
+  private securityValues!: SecurityValues;
+  private operationExamples!: OperationExample[];
+
+  public async setup(suiteConfig: SuiteConfig): Promise<void> {
+    await super.setup(suiteConfig);
+
+    this.server = suiteConfig.options.server;
+    this.schema = suiteConfig.schema;
```

Comment:
Why copy `schema` to a separate class-variable? I see it's still being used from the `suiteConfig` down on line 36

Reply:
The schema was moved.  The latter location was not updated by accident since it didn't report an error.

Reply:
It wouldn't report an error though because it still exists on `this.suiteConfig`. Do you think it's simpler if all this is copied out of suiteConfig to class-variables instead?

Reply:
I guess I just have a preference for short lines.  this.schema vs this.suiteConfig.schema

Reply:
Cool, makes sense to me!